wgw_version

```diff
@@ -0,0 +1 @@
+version = 4.42
```

Comment:
Change to 4.48

Reply:
version corrected and for simplicity removed the "version = " so you can immediately read the version number